inventories/opensearch/group_vars/all/all.yml

```diff
@@ -6,16 +6,12 @@ os_cluster_name: development-cluster
 os_download_url: https://artifacts.opensearch.org/releases/bundle/opensearch
 
 # opensearch version
-# 1.x Latest Version
-os_version: "1.3.2"
 # 2.x Latest Version
-# os_version: "2.0.0-rc1"
+os_version: "2.0.0"
 
 # opensearch dashboards version
-# 1.x Latest Version
-os_dashboards_version: "1.3.2"
 # 2.x Latest Version
-# os_dashboards_version: "2.0.0-rc1"
+os_dashboards_version: "2.0.0"
```

Comment:
I assume you will merge after tomorrows release?

Reply:
Same as above.